scripts/test-api-control

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+# API Control tests assert that the external third party APIs we are using
+# return data in an expected format. They query the relevant API with real (not
+# mocked) HTTP calls.
+#
+# As those tests are slow, and have a higher probability of flakiness because of
+# network issues or timeouts, we don't want to run them on each pre-commit hook
+# or CI commit. They can instead be run manually, or periodically from the CI.
+#
+# When it fails on the CI, it will generate a GitHub issue with the failure details
+# as well as a link to the run.
+
+# Running locally, with colors and live output
+if [ "$GITHUB_RUN_ID" = "" ]; then
+	jest \
+		./src/__tests__/api-control \
+		--forceExit \
+		--testPathIgnorePatterns=''
+	exit $?
+fi
+
+# Running on CI, creating an issue on failure
+echo "Wait while we run the tests"
+output=$(jest \
+	./src/__tests__/api-control \
+	--forceExit \
+	--testPathIgnorePatterns='' 2>&1)
```

Comment:
By default, jest redirects all its output to stderr. `2>&1` is here to redirect it back to stdout so we can capture it